internal/endpoint/mqtt.go

```diff
@@ -83,7 +83,9 @@ func (conn *MQTTConn) Send(msg string) error {
 			}
 			ops = ops.SetTLSConfig(&config)
 		}
-		ops = ops.SetClientID("tile38").AddBroker(uri)
+		nano := time.Now().UnixNano()
+		clientID := fmt.Sprintf("tile38_%x", nano) //the id of connected clients should be unique
```

Comment:
thoughts on using uuid vs timestamp?

Reply:
uuid would eliminate the risk of 2 connections from tile38, happening at exactly the same nanosecond, which is very unlikely.  Entire change of the client id to uuid (loosing the tile38_ prefx) is not good for the mqtt broker logs - it may have a lot of other clients - the admin needs filtering criteria. I can change it to tile38_<guid> ?